gem2rpm/scl.spec.erb

```diff
@@ -4,6 +4,7 @@ require 'json'
 root = `git rev-parse --show-toplevel`.strip
 SCL_PREFIXES = JSON.load(File.read(File.join(root, 'scl_prefixes.json')))
 LICENSES = JSON.load(File.read(File.join(root, 'licenses.json')))
+config.rules[:ignore].push(*['Guardfile'])
```

Comment:
@evgeni Added a commit that adds Guardfile to our ignore list from here on out.

Reply:
We should do that more often. There's more files like that that we often end up ignoring.

Reply:
Agreed. That's why I laid the groundwork.

Reply:
@ehelms could you update to the syntax Ewoud posted from the other template?

Reply:
The irony is, I started with the syntax and the voice in my head said why repeat yourself. >.< Updated.